TeamAmaze/AmazeFileManager
# OPEN_DOCUMENT and CREATE_DOCUMENT intents support

#### Extra information
It is assumed that the file selected/ to be saved are on the Android device locally. Haven't tested if user attempts to pick the file at remote location - or even not able to test at all - so beware.

#### Issue tracker   
Fixes #3932

#### Automatic tests
  
#### Manual tests
  
- Device: Pixel 2 emulator
- OS: Android 9

1. Install KeePassDX
2. Install build including this changeset
3. Open KeePassDX
4. Tap Create Database
5. DocumentsUI and Amaze should show up as choice for handling the action
6. Use Amaze to handle the action. Just Once or Always both are OK
7. Choose the destination folder where the new database file should be stored
8. Tap Save at the Snack Bar, file dialog should popup with keepass.kbdx filled in
9. Save as-is or enter a new filename, best end with .kbdx
10. Amaze should close and back to KeePassDX, fill in the details as usual
11. Close Amaze and KeePassDX
12. Reopen KeePassDX, tap Open Existing Database
13. DocumentsUI and Amaze should show up as choice for handling the action
14. Use Amaze to handle the action. Just Once or Always both are OK
15. Choose the database created at 9 by browsing to the folder where the file is located
16. Amaze should close and back to KeePassDX, fill in the credentials to unlock database
17. The database should open properly

#### Build tasks success
Successfully running following tasks on local:
